go/oasis-test-runner/scenario/e2e/registry_cli.go

```diff
@@ -642,6 +641,31 @@ func (sc *registryCLIImpl) testRuntime(ctx context.Context, childEnv *env.Env, c
 				},
 			},
 		},
+		Constraints: map[scheduler.CommitteeKind]map[scheduler.Role]registry.SchedulingConstraints{
+			scheduler.KindComputeExecutor: {
+				scheduler.RoleWorker: {
+					MinPoolSize: &registry.MinPoolSizeConstraint{
+						Limit: 1,
+					},
+					ValidatorSet: &registry.ValidatorSetConstraint{},
+				},
+				scheduler.RoleBackupWorker: {
+					MinPoolSize: &registry.MinPoolSizeConstraint{
+						Limit: 2,
+					},
+				},
+			},
+			scheduler.KindStorage: {
+				scheduler.RoleWorker: {
+					MinPoolSize: &registry.MinPoolSizeConstraint{
+						Limit: 9,
+					},
+					MaxNodes: &registry.MaxNodesConstraint{
+						Limit: 1,
```

Comment:
was this implicit before? or is this added

Reply:
This (MaxNodes) was not implicit. It is added here to test the CLI for generating runtime descriptors.

Reply:
ok